mappings/net/minecraft/nbt/NbtTagSizeAllocator.mapping

```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/class_2505 net/minecraft/nbt/NbtTagSizeAllocator
```

Comment:
This is a size tracker

Reply:
Will change. Still better than the old name though, who the heck named it `PositionTracker`?